camus-shopify/script/update_oozie_schedule

```diff
@@ -58,7 +58,7 @@ def main():
 
     # the coordinator handles the schedule
     starttime = datetime.now().strftime("%Y-%m-%dT%H:%MZ")
```

Comment:
I prefer that we hardcode this to half past the hour, else we can get very uneven distributed file sizes.